[Bug] Locale German isn't found

### 💻 Operating System

Windows

### 📦 Environment

Docker

### 🌐 Browser

Edge

### 🐛 Bug Description

When using "German" (not "German (Germany)") as default language in MS Edge, lobe-chat returns the error:

```
Error: Cannot find module './de.js'                                                                                                                                                                                                                     
    at /app/.next/server/chunks/69438.js:1:7617                                                                                                                                                                                                         
    at runNextTicks (node:internal/process/task_queues:60:5)                                                                                                                                                                                            
    at listOnTimeout (node:internal/timers:540:9)                                                                                                                                                                                                       
    at process.processTimers (node:internal/timers:514:7)                                                                                                                                                                                               
    at async H (/app/.next/server/chunks/69438.js:211:7731)                                                                                                                                                                                             
    at async eu (/app/.next/server/chunks/69438.js:211:8454) {                                                                                                                                                                                          
  code: 'MODULE_NOT_FOUND'                                                                                                                                                                                                                              
}   
```

### 🚦 Expected Behavior

_No response_

### 📷 Recurrence Steps

Set the default language to "German", visit the site (for me it works on first load) - second load of website it always throws an error. 

### 📝 Additional Information

Adding:

```typescript
export const normalizeLocale = (locale?: string) => {
  if (!locale) return 'en-US';

  switch (locale) {
    case 'zh-CN':
    case 'zh': {
      return 'zh-CN';
    }
    
// ---------NEW------------
    case 'de': {
      return 'de-DE';
    }
// ------------------------
    case 'en': {
      return 'en-US';
    }

    default: {
      return locale;
    }
  }
};
```
to /src/locales/resources.ts - fixes the problem

--- a/src/locales/resources.ts
+++ b/src/locales/resources.ts
@@ -28,6 +28,10 @@ export const normalizeLocale = (locale?: string) => {
       return 'zh-CN';
     }
 
+    case 'de': {
+      return 'de-DE';
+    }
+
     case 'en': {
       return 'en-US';
     }

